core/ustring.cpp

```diff
@@ -120,9 +120,13 @@ void String::copy_from(const char *p_cstr) {
 void String::copy_from(const CharType *p_cstr, int p_clip_to) {
 
 	int len = 0;
-	const CharType *ptr = p_cstr;
-	while (*(ptr++) != 0)
-		len++;
+
+	if (p_cstr) {
```

Comment:
Maybe it would be better to move the test at the top and add an early `resize(0); return;`? (+ maybe an error message?)